conjurapi/client.go

```diff
@@ -213,6 +213,19 @@ func (c *Client) SubmitRequest(req *http.Request) (resp *http.Response, err erro
 	return
 }
 
+func (c *Client) SubmitRequestWithCustomAuth(req *http.Request, token string) (resp *http.Response, err error) {
```

Comment:
I think we should make this private. Let's also have `SubmitRequest` use `SubmitRequestWithCustomAuth` instead of duplicating logic

Reply:
Good suggestion, thanks.